draft-irtf-nwcrg-coding-and-congestion-08.xml

```diff
@@ -503,17 +503,18 @@
 				<t>Research question 3 : "Should we quantify the harm that a coded flow would induce on a non-coded flow ? How can this be reduced while still benefiting from advantages brought by FEC ?"</t>
-				<t>Research question 4 : "If transport and FEC senders are not collocated, if the FEC is applied only on the last mile, would this raise fairness issues ?"</t>
+				<t>Research question 4 : "If transport and FEC senders are collocated and close to the client and applied only on the last mile, e.g. on a poor wireless link, would this raise fairness issues ?"</t>
 				<t>Research question 5 : "Should we propose a generic API to allow dynamic interactions between a transport protocol and a coding scheme ? This should consider existing APIs between application and transport layers."</t>
 				</section>
 
 			</section>
 
-			<section title="Advice for evaluating coding mechanisms">
-				<t>Our advice is that new research contributions should be mapped following the organization of this document. Otherwise, this may lead to wrong assumptions on the validity of the proposal and wrong ideas about the relevance of coding for a given use case.</t>
-				<t>The discussion provided in this document aims to encourage the research community to also consider congestion control aspects when proposing and comparing FEC coding solutions in communication systems. As one example, this draft proposes discussions on the impact of the proposed FEC solution on congestion control, especially loss-based congestion control mechanisms. When a research work aims at improving throughput by hiding the packet loss signal from congestion control, the authors should 1) discuss the advantages of using the proposed FEC solution compared to replacing the congestion control by one that ignores a portion of the encountered losses, 2) critically discuss the impact of hiding packet loss from the congestion control mechanism.</t>
+			<section title="Recommandations and advice for evaluating coding mechanisms">
+				<t>Research Recommandation 1: From a congestion control point-of-view, a repaired packet must be considered as a lost packet. This does not apply to the usage of FEC on a path that is known to be lossy.</t>
+				<t>Research Recommandation 2: New research contributions should be mapped following the organization of this document (above, below, in the congestion control) and should consider congestion control aspects when proposing and comparing FEC coding solutions in communication systems.</t>
+				<t>Research Recommandation 3: When a research work aims at improving throughput by hiding the packet loss signal from congestion control, the authors should 1) discuss the advantages of using the proposed FEC solution compared to replacing the congestion control by one that ignores a portion of the encountered losses, 2) critically discuss the impact of hiding packet loss from the congestion control mechanism. </t>
```

Comment:
```suggestion
				<t>Research Recommendation 3: When a research work aims at improving throughput by hiding the packet loss signal from congestion control (e.g., because the path between the sender and receiver is known to consist of a noisy wireless link), the authors should 1) discuss the advantages of using the proposed FEC solution compared to replacing the congestion control by one that ignores a portion of the encountered losses, 2) critically discuss the impact of hiding packet loss from the congestion control mechanism. </t>
```